woocommerce/woocommerce
# Add tracks to new product and edit product screens

### All Submissions:






### Changes proposed in this Pull Request:

Adds tracks to the new product and edit product screens to help better understand how the UI is being used.

Closes #32672  .

### How to test the changes in this Pull Request:

1. Edit an existing product.
2. Walk through the events in https://github.com/woocommerce/woocommerce/issues/32672 making sure all are triggered with the correct props (note that "save" events have been bundled into one per p1652906572219459-slack ).
3. Click Products->Add new
4. Repeat these event steps prior to saving the product
5. Navigate to the "Products" list page and make sure no regressions have occurred with the events there. https://github.com/woocommerce/woocommerce/pull/32949

### Other information:




### FOR PR REVIEWER ONLY:
